### PR TITLE
MINOR: Improve JavaDocs of KafkaStreams `context.commit()`

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

File: streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentTestUtils.java [hunk 1]
```diff
@@ -805,12 +805,14 @@ static Map<Subtopology, Set<TaskId>> getTopologyGroupTaskMap() {
         return Collections.singletonMap(SUBTOPOLOGY_0, Collections.singleton(new TaskId(1, 1)));
     }
 
-    static void verifyStandbySatisfyRackReplica(final Set<TaskId> taskIds,
+    static void verifyStandbySatisfyRackReplica(
+        final Set<TaskId> taskIds,
```
Comment (mjsax) [inline streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentTestUtils.java:809]: Unrelated side cleanup to fix formatting

**Review by bbejeck — APPROVED** [review]
LGTM
